modules/geth/charts.go

```diff
@@ -0,0 +1,143 @@
+package geth
+
+import "github.com/netdata/go.d.plugin/agent/module"
+
+type (
+	Charts = module.Charts
+	Chart  = module.Chart
+	Dims   = module.Dims
+	Dim    = module.Dim
+)
+
+var charts = Charts{
+	chartAncientChainData.Copy(),
+	chartChaidataDisk.Copy(),
+	chartTxPoolPending.Copy(),
+	chartBlockProcessingTime.Copy(),
+	chartTxPoolQueued.Copy(),
+	chartP2PNetwork.Copy(),
+	chartP2PNetworkDetails.Copy(),
+	chartNumberOfPeers.Copy(),
+	chartRpcInformation.Copy(),
+}
+
+var (
+
+	chartAncientChainData = Chart{
+		ID:    "chaindata_ancient",
+		Title: "Ancient Chaindata",
+		Units: "bytes",
+		Fam:   "chaindata",
+		Ctx:   "geth.chaindata",
+		Dims: Dims{
+			{ID:ethDbChainDataAncientRread , Name: "Ancient chaindata reads"},
+			{ID: ethDbChainDataAncientWrite, Name: "Ancient chaindata writes", Mul: -1},
+		},
+		
+	}
+	chartChaidataDisk = Chart{
+		ID:    "chaindata_disk",
+		Title: "Chaindata on disk",
+		Units: "bytes",
+		Fam:   "chaindata",
+		Ctx:   "geth.chaindata_disk",
+		Dims: Dims{
+			{ID:ethDbChaindataDiskRead , Name: "Disk Chaindata reads"},
+			{ID: ethDbChainDataDiskWrite, Name: "Disk Chaindata writes", Mul: -1},
+		},
+	}
+	chartBlockProcessingTime = Chart{
+		ID: "blockProcessing_time", 
+		Title: "Block processing time",
+		Units: "seconds",
+		Fam: "block_processing", 
+		Ctx: "geth.block_processing",
+		Dims: Dims{
+			{ID: blockProcessing, Name: "Block processing Time"},
```

Comment:
<details>
<summary>les_server_blockProcessingTime is summary metric</summary>

```cmd
# TYPE les_server_blockProcessingTime summary
les_server_blockProcessingTime {quantile="0.5"} 0
les_server_blockProcessingTime {quantile="0.75"} 0
les_server_blockProcessingTime {quantile="0.95"} 0
les_server_blockProcessingTime {quantile="0.99"} 0
les_server_blockProcessingTime {quantile="0.999"} 0
les_server_blockProcessingTime {quantile="0.9999"} 0
```

</details>

How do you plan to represent it in a chart with one dimension?